lib/src/user_agent_application.dart

```diff
@@ -71,16 +69,13 @@ class UserAgentApplication {
   Future<AuthResponse> acquireTokenPopup(AuthRequest request) async {
     final response = await _convertMsalPromise<interop.AuthResponse>(
         _callJsMethod(() => _jsObject.acquireTokenPopup(request._jsObject)));
-
     return AuthResponse._fromJsObject(response);
   }
 
   /// Acquires an access token by redirecting the user to the authorization endpoint.
   ///
   /// To renew an ID token, pass the client ID as the only scope in the [request].
-  void acquireTokenRedirect(AuthRequest request) {
-    _callJsMethod(() => _jsObject.acquireTokenRedirect(request._jsObject));
-  }
+  void acquireTokenRedirect(AuthRequest request) => _callJsMethod(() => _jsObject.acquireTokenRedirect(request._jsObject));
```

Comment:
This should keep the bracket syntax especially since the expression syntax puts this line way over the 80 character limit.